<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CyberGod</class>
 <widget class="QMainWindow" name="CyberGod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CyberGod</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/banners/banner.png&quot; height=400
/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="cyberGod_Tabs">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="tabsClosable">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="scanTab">
         <attribute name="icon">
          <iconset resource="img.qrc">
           <normaloff>:/menu_icons/scan_menu.png</normaloff>:/menu_icons/scan_menu.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Scan</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="scanTab_DriveLabel">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Drive:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="scanTab_DriveComboBox"/>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="scanTab_scanTypeLabel">
              <property name="text">
               <string>Scan Type:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="scanTab_scanTypeComboBox"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="scanTab_ScanButton">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>START SCAN</string>
              </property>
              <property name="icon">
               <iconset resource="img.qrc">
                <normaloff>:/buttons/scan_tab_scan_button.png</normaloff>:/buttons/scan_tab_scan_button.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="scanTab_currentlyScanningLabel">
              <property name="text">
               <string>Currently Scanning:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="scanTab_currentlyScanningFileLocation">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="scanTab_scannedFilesLabel">
              <property name="text">
               <string>Scanned Files:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="scanTab_ScannedFilesCount">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="scanTab_scanResultTable">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <column>
             <property name="text">
              <string>Threat Name</string>
             </property>
             <property name="icon">
              <iconset resource="img.qrc">
               <normaloff>:/tables/threat.png</normaloff>:/tables/threat.png</iconset>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Location</string>
             </property>
             <property name="icon">
              <iconset resource="img.qrc">
               <normaloff>:/tables/threat_location.png</normaloff>:/tables/threat_location.png</iconset>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Ignore</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="scanTab_deleteButton">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>Qurantine</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="usbTab">
         <attribute name="icon">
          <iconset resource="img.qrc">
           <normaloff>:/menu_icons/usb_menu.png</normaloff>:/menu_icons/usb_menu.png</iconset>
         </attribute>
         <attribute name="title">
          <string>USB</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Drive:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="usbTab_usbDriveComboBox"/>
            </item>
            <item>
             <widget class="QPushButton" name="usbTab_scanButton">
              <property name="text">
               <string>START SCAN</string>
              </property>
              <property name="icon">
               <iconset resource="img.qrc">
                <normaloff>:/buttons/scan_tab_scan_button.png</normaloff>:/buttons/scan_tab_scan_button.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Auto-run file:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="usbTab_autoRunFileLabel">
              <property name="text">
               <string>NA</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Scanning:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="usbTab_currentlyScannigFileLabel">
              <property name="text">
               <string>NA</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>File executed by autorun:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="usbTab_fileExecutedByAutorunLabel">
              <property name="text">
               <string>NA</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Scanned:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="usbTab_totalFilesScannedLabel">
              <property name="text">
               <string>NA</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Auto-run file trace in system:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="usbTab_autorunTraceTable">
              <column>
               <property name="text">
                <string>Location</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Threat Name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string/>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Scan Result:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="usbTab_scanResultTable">
              <column>
               <property name="text">
                <string>Threat Name</string>
               </property>
               <property name="icon">
                <iconset resource="img.qrc">
                 <normaloff>:/tables/threat.png</normaloff>:/tables/threat.png</iconset>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Location</string>
               </property>
               <property name="icon">
                <iconset resource="img.qrc">
                 <normaloff>:/tables/threat_location.png</normaloff>:/tables/threat_location.png</iconset>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;img src=&quot;:/banners/usb_banner.png&quot; height=60 width=600/&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="icon">
          <iconset resource="img.qrc">
           <normaloff>:/buttons/clean_brush.png</normaloff>:/buttons/clean_brush.png</iconset>
         </attribute>
         <attribute name="title">
          <string>DupeClean</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#550000;&quot;&gt;Duplicate Files Remover&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;img height=150 width=150 src=&quot;:/tabs/duplicate.png&quot;/&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <spacer name="horizontalSpacer_8">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox">
              <item>
               <property name="text">
                <string>DRIVE</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>FOLDER</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>SCAN FOR DUPLICATES</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_11">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Currently Scanning: </string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>NA</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Duplicates Found:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>NA</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="icon">
          <iconset resource="img.qrc">
           <normaloff>:/menu_icons/update.png</normaloff>:/menu_icons/update.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Update</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <spacer name="horizontalSpacer_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;img height=100 width=100 src=&quot;:/menu_icons/update.png&quot;/&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton">
                  <property name="text">
                   <string>UPDATE</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QLabel" name="label_16">
                  <property name="text">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#0055ff;&quot;&gt;Database Updater&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QFormLayout" name="formLayout">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_17">
                    <property name="text">
                     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Database Version:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLabel" name="label_18">
                    <property name="text">
                     <string>NA</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_19">
                    <property name="text">
                     <string>Last-Updated:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLabel" name="label_20">
                    <property name="text">
                     <string>NA</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_10">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuScan">
    <property name="title">
     <string>Scan</string>
    </property>
    <addaction name="actionCustom_Scan"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Information</string>
    </property>
    <addaction name="actionAbout_us"/>
    <addaction name="actionThird_party_notices"/>
    <addaction name="actionLicense_Agreenment"/>
   </widget>
   <widget class="QMenu" name="menuTools_2">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuUSB_Tools">
     <property name="title">
      <string>USB Tools</string>
     </property>
     <addaction name="actionFix_shortcut_virus"/>
     <addaction name="actionDetect_Rubber_Ducky"/>
    </widget>
    <addaction name="menuUSB_Tools"/>
   </widget>
   <addaction name="menuScan"/>
   <addaction name="menuTools_2"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCustom_Scan">
   <property name="text">
    <string>Custom Scan</string>
   </property>
  </action>
  <action name="actionAbout_us">
   <property name="text">
    <string>About us</string>
   </property>
  </action>
  <action name="actionFix_shortcut_virus">
   <property name="text">
    <string>Fix shortcut virus</string>
   </property>
  </action>
  <action name="actionDetect_Rubber_Ducky">
   <property name="text">
    <string>Detect Rubber Ducky</string>
   </property>
  </action>
  <action name="actionPC_Info">
   <property name="text">
    <string>PC Info</string>
   </property>
  </action>
  <action name="actionOpenDB">
   <property name="text">
    <string>Update Database</string>
   </property>
  </action>
  <action name="actionSupport_this_project">
   <property name="text">
    <string>Support this project</string>
   </property>
  </action>
  <action name="actionThird_party_notices">
   <property name="text">
    <string>Third party notices</string>
   </property>
  </action>
  <action name="actionLicense_Agreenment">
   <property name="text">
    <string>License Agreenment</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
